ts/features/itwallet/issuance/screens/ItwIssuanceCredentialPreviewScreen.tsx

```diff
@@ -1,29 +1,61 @@
+import { ContentWrapper } from "@pagopa/io-app-design-system";
 import * as O from "fp-ts/lib/Option";
 import { pipe } from "fp-ts/lib/function";
 import React from "react";
 import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import { FooterActions } from "../../../../components/ui/FooterActions";
 import { IOScrollViewWithLargeHeader } from "../../../../components/ui/IOScrollViewWithLargeHeader";
 import I18n from "../../../../i18n";
 import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import ROUTES from "../../../../navigation/routes";
+import { identificationRequest } from "../../../../store/actions/identification";
+import { useIODispatch } from "../../../../store/hooks";
+import { ItwCredentialClaimsList } from "../../common/components/ItwCredentialClaimList";
+import { useItwDismissalDialog } from "../../common/hooks/useItwDismissalDialog";
 import {
   ItWalletError,
   getItwGenericMappedError
 } from "../../common/utils/itwErrorsUtils";
 import { ItwCredentialsMocks } from "../../common/utils/itwMocksUtils";
 import { StoredCredential } from "../../common/utils/itwTypesUtils";
-import { ITW_ROUTES } from "../../navigation/routes";
-import { ItwCredentialPreviewScreenContent } from "../components/ItwCredentialPreviewScreenContent";
 
 export const ItwIssuanceCredentialPreviewScreen = () => {
   const navigation = useIONavigation();
+  const dispatch = useIODispatch();
   const credentialOption = O.some(ItwCredentialsMocks.mdl);
+  const dismissDialog = useItwDismissalDialog();
```

Comment:
There's an error when going back from the mDL preview screen. That's probably because the screen is currently isolated. Not an issue at the moment but it could crash the app in production if someone tries that screen.

Reply:
There is an extra `pop` in the dismiss handler, which is not necessary in the playground. I've removed it so it doesn't break anymore. We will refine the navigation behavior once we get to the final steps of the flow.
You can see the change here: https://github.com/pagopa/io-app/pull/5910/commits/208c130338c229d8a707fc65c6c3ca645c4535c5